.github/checklist.yml

```diff
@@ -0,0 +1,10 @@
+paths:
+  "Cargo.lock":
+    - Did you review changed dependencies for security concerns?
```

Comment:
We should be running `cargo audit` on CI

Reply:
Good point. I can modify. I still think dependency changes should be reviewed by 3 or more team members since this can be a risky change.

Reply:
Recommend only explicit changes in the `cargo.toml`.